js8call_integration.py

```
@@ -92,41 +93,28 @@ def create_tables(self):
+        allowed_tables = {
+            'messages': ('receiver', 'messages'),
+            'groups': ('groupname', 'groups'),
+            'urgent': ('groupname', 'urgent')
+        }
+
+        if table not in allowed_tables:
+            self.logger.error(f"Attempted access to invalid/unauthorized table: {table}")
+            return
+
+        col_name, real_table = allowed_tables[table]
+        sql = f"INSERT INTO {real_table} (sender, {col_name}, message) VALUES (?, ?, ?)"
```

Ruff (0.15.1)

[error] Possible SQL injection vector through string-based query construction

(S608)